extras/docker/Dockerfile

```diff
@@ -1,9 +1,44 @@
-#Download base image ubuntu 16.04
-FROM mauriciojost/scala:latest
- 
+# REMARK: https://github.com/hseeberger/scala-sbt/blob/master/Dockerfile
```

Comment:
You're the docker expert.

Reply:
There's nothing wrong with your images and dockerfiles, the change is just bureaucracy to make this build cleanly on ARM cpus